pkg/kn/root/root.go

```diff
@@ -88,6 +89,7 @@ func NewRootCommand() (*cobra.Command, error) {
 		{
 			Header: "Eventing Commands:",
 			Commands: []*cobra.Command{
+				broker.NewBrokerCommand(p),
```

Comment:
maybe you can move it below `source` so that broker & trigger are close together. Or do we want to keep it alphabetical ? not sure.

Reply:
No strong opinion here. I went for alphabetical order, but your suggestion sounds very reasonable.

Reply:
Ok, then lets move sources one up so that broker & trigger are together.

Reply:
Fixed in https://github.com/knative/client/pull/894/commits/4e70573e152df0e9e9055ccc67f958f5a63a30f1.